packages/core/src/electron-browser/menu/electron-menu-style.css

```diff
@@ -0,0 +1,87 @@
+/********************************************************************************
+ * Copyright (C) 2021 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+#theia-drag-panel {
+  position: absolute;
+  display: block;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: calc(100% - 4px);
+  margin: 4px;
+  -webkit-app-region: drag !important;
+}
+
+#theia-top-panel > * {
+  -webkit-app-region: no-drag;
+}
+
+#window-controls {
+  display: grid;
+  grid-template-columns: repeat(3, 48px);
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 100%;
+}
+
+#window-controls .control-button {
+  grid-row: 1 / span 1;
+  display: flex;
+  line-height: 30px;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+#minimize-button {
+  grid-column: 1;
+}
+#maximize-button, #restore-button {
+  grid-column: 2;
+}
+#close-button {
+  grid-column: 3;
+}
+
+#window-controls .control-button {
+  user-select: none;
+}
+
+#window-controls .control-button:hover {
+  background: rgba(50%, 50%, 50%, 0.2);
+}
+
+#close-button:hover {
+  background: #E81123 !important;
+}
+
+#close-button:hover:before {
+  color: white;
+}
+
+#restore-button {
+  display: none !important;
+}
+
+.maximized #restore-button {
+  display: flex !important;
+}
```

Comment:
Are these `!important` specifications necessary? If possible I'd prefer something like `.not-maximized #restore-button { display: none; }` vs. `.maximized #restore-button { display: flex; }`, for example.

Reply:
Removing `!important` has rendered the `#close-button:hover` (ID + pseudoclass (0,1,1,0) specificity) rule inoperative, because it's competing with `#window-controls .control-button:hover` (ID + class + pseudoclass (0,1,2,0)). You could change it to `#window-controls #close-button:hover` (0,2,1,0) or change `window-controls` to a class rather than an ID, since `.window-controls .control-button:hover` would be have 0,0,3,0 specificity. Our CSS isn't very standardized or especially well organized, so you can choose how you'd like to resolve it. In general, I shy away from ID-scoped styling and `!important`, if they can be avoided.

Reply:
@msujew, just this minor thing to fix, and then I think this is good to go.

Reply:
@colin-grant-work Thanks, didn't notice that :)

I think that should be everything.